config/brakeman.ignore

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
file này là sao?

Reply:
Em chạy ci thì có cảnh báo file access nó bảo là phải đảm bảo các đường dẫn là an toàn. Em viết danh sách các đường dẫn an toàn rồi nên em cho cảnh báo này vào ignore của brakeman ạ. Cái ignore này là chạy lệnh nó tự tạo cho mình ạ